internal/suggestions/suggestion.go

```diff
@@ -5,12 +5,23 @@ import (
 	"github.com/speakeasy-api/sdk-generation-action/internal/cli"
 	"github.com/speakeasy-api/sdk-generation-action/internal/environment"
 	"github.com/speakeasy-api/sdk-generation-action/internal/git"
+	"github.com/speakeasy-api/sdk-generation-action/internal/logging"
+	"regexp"
+	"strconv"
 	"strings"
 )
 
-type prBodyInfo struct {
+var (
+	fileNameRegex      = regexp.MustCompile(`^(.*?(\.yaml|\.yml|\.json))`)
+	outputFileRegex    = regexp.MustCompile(`Suggestions applied and written to (.+)`)
```

Comment:
I think this line is only written if parallelization (or autonext or whatever) is enabled? Is there a weird case where we accidentally change that and break this?

Reply:
Works in `--serial`, but yes this line isn't written when auto-continue is disabled. That should be fine as this is action code, and we'd always run the CLI with it enabled in this context.